### DISCUSSION TITLE
1.4.1

### DISCUSSION THREAD

**Discussion by ivan-aksamentov** [nextstrain/nextclade#563] [Releases]
### [Fix] Format of CSV/TSV output files

We fixed a few mistakes in CSV and TSV output files, such as missing last delimiter when the "errors" column is empty, inconsistent application of quotation marks and incorrect numeric formats - decimals when integers should be.

<hr /><em>This discussion was created from the release <a href='https://github.com/nextstrain/nextclade/releases/tag/1.4.1'>1.4.1</a>.</em>

**Comment by geomen888** [comment]


**Reply by corneliusroemer** [reply]
Hey Eugene, how would you like to help?
